cowprotocol/cowswap
# feat(twap): pass order class and slippage to app-data

# Summary

Added a slippage value and `orderClass=twap` to `appData`


  # To Test

1. Create a TWAP order
2. Wait until the first part is created
- [ ] Part order's app-data should contain orderClass=twap
- [ ] Part order's app-data slippage should respect the entered value
